clients/cobol-lsp-vscode-extension/src/extension.ts

```diff
@@ -521,6 +526,26 @@ function registerCommands(context: vscode.ExtensionContext) {
       },
     ),
   );
+  context.subscriptions.push(
```

Comment:
registering the commands via `registerTextEditorCommand` should give you the check for active editor and edit session in the callback.